### PR TITLE
added kotlin

### PR DESCRIPTION
**Pull request by realjoni17**

### CODE REVIEW

File: README.md
Comment (andrewhoyer) [inline README.md:1]: The list in the main README is in alphabetical order. Please put Kotlin below JavaScript.

File: kotlin/info.md
Comment (andrewhoyer) [inline kotlin/info.md:1]: This file should be named README.md.

**Comment by anupdsouza** [comment]
@realjoni17 , the Kotlin folder should be renamed to use uppercased 'K', else it breaks the link in README.md

**Comment by realjoni17** [comment]
Its already in uppercase

**Review by andrewhoyer — APPROVED** [review]
Thanks for making the change. I think this is ready to go.

**Comment by andrewhoyer** [comment]
Hey @realjoni17. I didn't see the reply by @anupdsouza before. The comment is correct, the case of the actual folder matters. Right now the folder is "kotlin" but the README points to "Kotlin". The Capital option is better. Having it non-matching does make it fail. Consider this:

### Works
https://github.com/andrewhoyer/leap-year/tree/main/Bash

### Doesn't work
https://github.com/andrewhoyer/leap-year/tree/main/bash

So update the folder to Kotlin, and I'll then merge it into the repo.

**Comment by realjoni17** [comment]
i fixed it

File: README.md
Comment (andrewhoyer) [reply README.md:1]: Noting this one again. The Kotlin line should be in alphabetical order, under JavaScript.

**Comment by andrewhoyer** [comment]
Hi @realjoni17. I'm still seeing Kotlin listed last in the main README.md file. Can you edit it so it appears below JavaScript?

**Review by andrewhoyer — APPROVED** [review]

**Pull request merged**